nupic/research/frameworks/vernon/mixins/vary_batch_size.py

```diff
@@ -33,66 +33,39 @@ class VaryBatchSize(object):
     """
 
     def setup_experiment(self, config):
-
-        super().setup_experiment(config)
-
         # Get and validate batch sizes.
         batch_sizes = config.get("batch_sizes", None)
-        assert isinstance(batch_sizes, list), "Must specify list of batch sizes"
-        self.batch_sizes = batch_sizes
+        assert isinstance(batch_sizes, Sequence), "Must specify list of batch sizes"
```

Comment:
The comment should now read "Must specify a sequence of batch sizes"

Reply:
Ok, I can change it. (Often in code comments I use "list" in the generic English sense. Usually I rely on duck-typing rather than type checking -- as long as they pass in something that supports indexing and len, the code will work)

Reply:
I see. That's fair. Either way.